src/IdentityServer/Models/BackchannelUserLoginRequest.cs

```diff
@@ -59,4 +60,10 @@ public class BackchannelUserLoginRequest
     /// Gets or sets the validated resources.
     /// </summary>
     public ResourceValidationResult ValidatedResources { get; set; } = default!;
+
+    /// <summary> 
+    /// Gets or sets a dictionary of custom properties that can pass extra state
+    /// to the notification process.
+    /// </summary>
+    public Dictionary<string, object> Context { get; set; } = new();
```

Comment:
I would change this to `Properties`. I feel **Context** should be reserved for a property containing a context object.